test/parallel/test-http-server-keepalive-req-gc.js

```diff
@@ -16,7 +16,7 @@ if (common.isWindows) {
 
 let client;
 const server = createServer(common.mustCall((req, res) => {
-  onGC(req, { ongc: common.mustCall() });
+  onGC(req, { ongc: () => { server.close(); } });
```

Comment:
The `common.mustCall()` bit is the main part of the test here :)

```suggestion
  onGC(req, { ongc: common.mustCall(() => { server.close(); }) });
```